app/views/public/_eventstatus.html.erb

```diff
@@ -0,0 +1,13 @@
+<% @user = current_user %>
+<% @event = event %>
```

Comment:
You don't need those assignments, `current_user` is a global helper and `event` is a variable that should come from the view that calls this partial.